docs/ldml/tr35.html

```diff
@@ -2369,6 +2369,29 @@ <h4><a href="#Key_And_Type_Definitions_" name=
           <em>Unknown or Invalid Currency</em>.</p>
         </td>
       </tr>
+      <tr class="changed">
+        <td colspan="4"><strong>A <a href=
+        "#UnicodeDictionaryBreakExclusionIdentifier" name=
+        "UnicodeDictionaryBreakExclusionIdentifier" id=
+        "UnicodeDictionaryBreakExclusionIdentifier">Unicode Dictionary Break Exclusion Identifier</a>
+        specifies scripts to be excluded from dictionary-based text break (for words and lines).
+        The valid values are of one or more items of type SCRIPT_CODE as specified in the
+        <em>name</em> attribute value in the <em>type</em>
+        element of key name="dx" in bcp47/<a target="_blank" href=
+        "https://github.com/unicode-org/cldr/tree/latest/common/bcp47/segmentation.xml">segmentation.xml</a>.</strong></td>
+      </tr>
+      <tr class="changed">
+        <td>"dx"</td>
+        <td>Dictionary break script exclusions</td>
+        <td>
+          <i>ISO 15924 script code</i>
+        </td>
+        <td>
+          <p>Items of type SCRIPT_CODE, which are codes consisting of 4 ASCII letters
```

Comment:
Change this to reference a [valid Unicode script subtag](http://unicode.org/reports/tr35/#unicode_script_subtag)

(maybe also other places if you have time)

Reply:
Thanks, I did make that change